ambry-mysql/src/main/java/com/github/ambry/accountstats/AccountStatsMySqlStore.java

```diff
@@ -136,8 +143,16 @@ public AccountStatsMySqlStore(List<MySqlUtils.DbEndpoint> dbEndpoints, String lo
   public void storeStats(StatsWrapper statsWrapper) {
     StatsSnapshot prevSnapshot =
         previousStats == null ? new StatsSnapshot((long) -1, new HashMap<>()) : previousStats.getSnapshot();
-    applyFunctionToContainerUsageInDifferentStatsSnapshots(statsWrapper.getSnapshot(), prevSnapshot,
-        this::tryUpdateStorageUsage);
+    ContainerUsageFunction function = this::tryUpdateStorageUsage;
+    ContainerStorageFunctionBatchUpdater batchUpdater = null;
+    if (batchEnabled) {
+      batchUpdater = new ContainerStorageFunctionBatchUpdater();
```

Comment:
Would it be better to catch SQLException here (or declare in method signature) than to eat exceptions and march on?